src/addLiquidity.ts

```diff
@@ -0,0 +1,127 @@
+import { Pool, StableswapPoolParams } from "./pool";
+import {
+  StableswapCalculator,
+  getAddLiquidityBonusPct,
+} from "./stableswapCalculator";
+import { TransactionGroup } from "./transactionGroup";
+
+/**
+ * The effect of adding liquidity to the pool.
+ */
+export type AddLiquidityEffect = {
+  /**
+   * Amount of new liquidity tokens minted when adding the liquidity. All the minted tokens will be received by the liquidity provider.
```

Comment:
as a reminder, because we talked about it - not all are received on first add liq